README.md

```diff
@@ -61,10 +61,10 @@ optimal parameter configurations to use during model compilation.
 
 ### Models
 
-Model name | Model recipes | Serving apps
----------- | ------------- | ------------
-SDXL       | [`sharktank/sharktank/models/punet/`](https://github.com/nod-ai/shark-ai/tree/main/sharktank/sharktank/models/punet) | [`shortfin/python/shortfin_apps/sd/`](https://github.com/nod-ai/shark-ai/tree/main/shortfin/python/shortfin_apps/sd)
-llama      | [`sharktank/sharktank/models/llama/`](https://github.com/nod-ai/shark-ai/tree/main/sharktank/sharktank/models/llama) | [`shortfin/python/shortfin_apps/llm/`](https://github.com/nod-ai/shark-ai/tree/main/shortfin/python/shortfin_apps/llm)
+Model name | Model recipes | Serving apps | Guide |
+---------- | ------------- | ------------ | ----- |
```

Comment:
Ah, this table is a bit too wide for GitHub's standard view, it has a scroll bar: 
![image](https://github.com/user-attachments/assets/fb370d50-c833-4987-90e8-452be0332060)